lib/webmock/fixtures/version.rb

```diff
@@ -0,0 +1,6 @@
+module WebMock
+  # WebMock fixture helper
+  module Fixtures
+    VERSION = "0.1.0"
```

Comment:
Yeah, it does seem a little weird, but this seems to be what a bunch of other gems do. `WebMock`, `factory_girl`, `rspec-core`, `HTTParty`.